x-pack/legacy/plugins/siem/public/components/page/network/network_top_n_flow_table/columns.tsx

```diff
@@ -38,6 +38,187 @@ export type NetworkTopNFlowColumns = [
   Columns<NetworkTopNFlowEdges>
 ];
 
+export type NetworkTopNFlowColumnsIpDetails = [
+  Columns<NetworkTopNFlowEdges>,
+  Columns<NetworkTopNFlowEdges>,
+  Columns<NetworkTopNFlowEdges>,
+  Columns<TopNFlowNetworkEcsField['bytes_in']>,
+  Columns<TopNFlowNetworkEcsField['bytes_out']>,
+  Columns<NetworkTopNFlowEdges>
+];
+
+export const getNetworkTopNFlowColumnsIpDetails = (
```

Comment:
In effort to avoid duplicating all the columns again, it may be a good idea to follow a similar pattern as the `Authentication Process` table here:

https://github.com/elastic/kibana/blob/4ebcace93a16652b2c78a9a885bcef1eac7d193b/x-pack/legacy/plugins/siem/public/components/page/hosts/authentications_table/index.tsx#L337-L350